### DISCUSSION TITLE
C++ Bindings?

### DISCUSSION THREAD

**Discussion by Advik-B** [python-pillow/Pillow#6404] [Q&A] [answered]
How can I use this in a c++ app

**Comment by hugovk** [comment, marked as answer]
You can't, Pillow only has a public Python API.

See https://github.com/python-pillow/Pillow/issues/4532.

**Reply by Advik-B** [reply]
Is there any fork alternative?

**Reply by radarhere** [reply]
No, no one (that I'm aware of) has forked Pillow to have a public C interface.
